src/thermo/critProperties.xml

```diff
@@ -0,0 +1,6331 @@
+ 
+ These critical properties are used to calculate the a and b parameters 
+ for real gas equations of state, such as Redlich-Kwong and Peng-Robinson.
+ 
+ The a and b parameters can be provided explicitly by the user in the input
+ file.  In the case that they are not provided for a species, the function
+ initThermoXML will search this database for critical properties, which can
+ also be used to calculate the a and b coefficients.  If critical properties
+ are not found and coefficients are not provided, then an error is thrown.
+ 
+ These critical properties have been compiled from various sources by
+ C. Karakaya (https://orcid.org/0000-0001-5102-5476)    
+ -->
+
+<ctml>
+    
+	   <species name= "CCL4">
+           <ChemName name ="carbon-tetrachloride"> </ChemName>
+           <MW value = "153.823"> </MW>
+           <Tc value= "556.5"> </Tc>
+           <Pc value= "45.6E6 "> </Pc>
+           <Vc value="275.9"> </Vc>
+           <Zc value="0.272"> </Zc>
+           <omega value="0.193"> </omega>
+       </species>
+       <species name= "CCL3F">
+           <ChemName name ="trichlorofluoromethane"> </ChemName>
+           <MW value = "137.368"> </MW>
+           <Tc value= "471.2"> </Tc>
+           <Pc value= "43.8E6 "> </Pc>
+           <Vc value="248"> </Vc>
+           <Zc value="0.28"> </Zc>
+           <omega value="0.189"> </omega>
+       </species>
+       <species name= "CCL2F2">
+           <ChemName name ="dichlorodifluoromethane"> </ChemName>
+           <MW value = "120.914"> </MW>
+           <Tc value= "384.8"> </Tc>
+           <Pc value= "40.1E6 "> </Pc>
+           <Vc value="217.6"> </Vc>
+           <Zc value="0.273"> </Zc>
+           <omega value="0.204"> </omega>
+       </species>
+       <species name= "CCLF3">
+           <ChemName name ="chlorotrifluoromethane"> </ChemName>
+           <MW value = "104.459"> </MW>
+           <Tc value= "302"> </Tc>
+           <Pc value= "39E6 "> </Pc>
+           <Vc value="180.3"> </Vc>
+           <Zc value="0.278"> </Zc>
+           <omega value="0.198"> </omega>
+       </species>
+       <species name= "CF4">
+           <ChemName name ="carbon-tetrafluoride"> </ChemName>
+           <MW value = "88.005"> </MW>
+           <Tc value= "227.5"> </Tc>
+           <Pc value= "37.4E6 "> </Pc>
+           <Vc value="140.5"> </Vc>
+           <Zc value="0.276"> </Zc>
+           <omega value="0.177"> </omega>
+       </species>
+       <species name= "CO">
+           <ChemName name ="carbon-monoxide"> </ChemName>
+           <MW value = "28.01"> </MW>
+           <Tc value= "133.4"> </Tc>
+           <Pc value= "35E6 "> </Pc>
+           <Vc value="93.1"> </Vc>
+           <Zc value="0.294"> </Zc>
+           <omega value="0.066"> </omega>
+       </species>
+       <species name= "CO2">
+           <ChemName name ="carbon-dioxide"> </ChemName>
+           <MW value = "44.01"> </MW>
+           <Tc value= "304.2"> </Tc>
+           <Pc value= "73.9E6 "> </Pc>
```

Comment:
I think all of these critical pressures are off by an order of magnitude -- the critical pressure for CO2 is 7.39 MPa, not 73.9 MPa.